Automattic/wp-calypso
# Open site assembler after selecting a virtual theme in the design picker

Fixes https://github.com/Automattic/wp-calypso/issues/76098

## Proposed Changes

Simplifies the selection of virtual themes in the design picker by omitting the design preview step and opening directly the site assembler.

**Before**

Step 1: Design picker | Step 2: Design preview | Step 3: Site assembler
--- | --- | ---
<img width="1154" alt="Screenshot 2023-05-08 at 15 12 22" src="https://user-images.githubusercontent.com/1233880/236833747-c585e9c5-813a-47a1-ad57-2ba19e0f717d.png"> | <img width="1154" alt="Screenshot 2023-05-08 at 15 12 32" src="https://user-images.githubusercontent.com/1233880/236833779-89a6ef53-8637-4f7b-92c8-b37636d80857.png"> | <img width="1155" alt="Screenshot 2023-05-08 at 15 12 40" src="https://user-images.githubusercontent.com/1233880/236833809-e5e1025c-d493-49ef-934a-4b1739d2feb3.png">

**After**

Step 1: Design picker | Step 2: Site assembler
--- | ---
<img width="1156" alt="Screenshot 2023-05-08 at 15 08 13" src="https://user-images.githubusercontent.com/1233880/236833891-3a5b3cb4-3234-43b1-b837-5d653a060f01.png"> | <img width="1155" alt="Screenshot 2023-05-08 at 15 08 24" src="https://user-images.githubusercontent.com/1233880/236833913-4e6112b3-2ce9-4838-9fe5-9abf5c328898.png">


## Testing Instructions

- Use the Calypso live link below.
- Go to `/start`.
- Follow the onboarding flow until you reach the "Pick a design" step.
- Select a virtual theme.
- Make sure the site assembler is opened. 
- Click on "Continue".
- Once in the launchpad, click on the "Select a design" task
- Select a regular theme now.
- Make sure you land on the design preview screen.
- Click on "Continue".
- Make sure it's activated as expected.
- Use the "Select a design" task again in the launchpad.
- Select a virtual theme now.
- Make sure it's activated without going through the site assembler (not sure if this is expected, but it's the current behavior in production as of https://github.com/Automattic/wp-calypso/pull/76813).
